src/js/App/Sidenav/Navigation.js

```diff
@@ -117,18 +135,13 @@ export const Navigation = ({ settings, activeApp, activeLocation, onNavigate, on
             onClick={(event, subItem) => (item.subItems ? onClick(event, subItem, item) : onClick(event, item))}
           />
         ))}
-        {extraLinks[activeLocation]?.map?.((item) => (
-          <NavItem
-            className="ins-c-navigation__additional-links"
-            key={item.id}
-            to={item.url || item.link}
-            ouiaId={item.id}
-            target="_blank"
-            rel="noopener noreferrer"
-          >
-            {item.title} {item.external && <ExternalLinkAltIcon />}
-          </NavItem>
-        ))}
+        {extraLinks[activeLocation]?.map?.((item) =>
+          item?.expandable && activeLocation === 'insights' ? (
+            <NavExpandable title={item.title}>{item?.subItems?.map((item) => navItem(item))}</NavExpandable>
```

Comment:
Can you convert this to a React Component. Uppercase and all that jazz. It makes a mess in virtual DOM when used as a function.

Reply:
good to know, updated @Hyperkid123 🤗